### PR TITLE
[frenemies] Migration logic

### PR DESCRIPTION
**Pull request by Jordan-Mysten**

### CODE REVIEW

File: dapps/frenemies/src/routes/Setup.tsx
Comment (amnn) [inline dapps/frenemies/src/routes/Setup.tsx:47]: Maybe a dumb question, but does ordering matter here? i.e. should we check for the account before the scorecard(s) so that we redirect to `/connect` before `/migrate`?

**Review by amnn — COMMENTED** [review]
This LGTM, let me know if this needs a stamp, but otherwise, I think someone more TS/React-literate should review.

File: dapps/frenemies/src/main.tsx
Comment (Jordan-Mysten) [reply dapps/frenemies/src/main.tsx:17]: This is being deleted in this PR, so they are being cleaned up 👀

File: dapps/frenemies/src/main.tsx
Comment (Jordan-Mysten) [reply dapps/frenemies/src/main.tsx:42]: Ditto as well, this diff is removal

File: dapps/frenemies/src/routes/Setup.tsx
Comment (Jordan-Mysten) [reply dapps/frenemies/src/routes/Setup.tsx:46]: It shouldn't, because these should are all technically mutually exclusive states.

Comment (Jordan-Mysten) [reply dapps/frenemies/src/routes/Setup.tsx:46]: Though I agree it's messy how this is written, ideally would not have as much of this inline and imperative like this

File: dapps/frenemies/src/main.tsx
Comment (amnn) [reply dapps/frenemies/src/main.tsx:17]: ... 🤦

**Review by mamos-mysten — APPROVED** [review]

**Pull request merged**
